src/btree/bt_vrfy.c

```diff
@@ -42,13 +42,15 @@ typedef struct {
     int verify_err;
 } WT_VSTUFF;
 
+static int __compare_page_id(const void *, const void *);
```

Comment:
Can we set the prefix "__verify" to be consistent?

Reply:
Good point, added prefix in [2d8dc56](https://github.com/wiredtiger/wiredtiger/pull/12419/commits/2d8dc566353a37e8a615a05e3d76cec7e56d2e14)